docker-cloud/apps/deploy-to-cloud-btn.md

```diff
@@ -21,6 +21,7 @@ The button redirects the user to the **Launch new Stack** wizard, with the stack
 
 The user can still modify the stack definition before deployment.
 
+Note: In order for the **Deploy to Docker Cloud** deployment to work correctly, users must be logged into Docker Cloud prior to clicking the button otherwise a 404 error will be returned.
```

Comment:
Can you add a `>` before this and enclose the "Note" in double-asterisks so it renders as a callout please?